Mapping/AuthzIdentityContext.cs

```diff
@@ -0,0 +1,57 @@
+using Aserto.AspNetCore.Middleware.Options;
+using Aserto.Authorizer.V2.API;
+using Google.Protobuf.WellKnownTypes;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+
+namespace Aserto.TodoApp.Mapping
+{
+    public sealed class AuthzIdentityContext
+    {
+        private static readonly AuthzIdentityContext instance = new AuthzIdentityContext();
+
+        static AuthzIdentityContext()
+        {
+
+        }
+        private AuthzIdentityContext()
+        {
+
+        }
+
+        public static AuthzIdentityContext Instance
+        {
+            get
+            {
+                return instance;
+            }
+        }
+
+        public Func<ClaimsPrincipal, IEnumerable<string>, IdentityContext> IdentityMapper { get; set; } = CustomIdentityContext;
+
+        private static IdentityContext CustomIdentityContext(ClaimsPrincipal identity, IEnumerable<string> supportedClaimTypes)
```

Comment:
This feels like logic that could be part of the SDK. How would it be different between applications?